build.sbt

```diff
@@ -226,11 +228,8 @@ lazy val storm = project.in(file("storm"))
 lazy val samza = project.in(file("samza"))
   .settings(commonSettings: _*)
   .settings(name := "tranquility-samza")
+  .settings(resolvers += "central-local" at "https://metamx.artifactoryonline.com/metamx/libs-releases-local")
```

Comment:
Can we use that instead then? I added similar ones in https://github.com/metamx/sumologic-log4j2-appender/blob/sumologic-log4j2-appender-2.5.0.13/pom.xml , but that was because I couldn't figure out how to do it in the maven release process.

I'm hoping to remove them in the near future.